public/js/products.js

```diff
@@ -19,4 +19,32 @@ function generateColumn(template, id, imgURL, Title) {
   $('#products-list').append(productList)
 }
 
-getProductResults()
+function reloadWebpageWithFilters() {
+  let params = new URLSearchParams(document.location.search)
```

Comment:
doesn't look like this variable is being used anywhere. I guess we can remove this?